tests/linearizability/watch.go

```diff
@@ -82,28 +82,37 @@ func validateWatchResponses(t *testing.T, responses [][]watchResponse, expectPro
 }
 
 func validateMemberWatchResponses(t *testing.T, responses []watchResponse, expectProgressNotify bool) {
-	var gotProgressNotify = false
-	var lastRevision int64 = 1
+	var (
+		gotProgressNotify       = false
+		lastEventRevision int64 = 1 // The event.Kv.ModRevision in the latest event.
+		lastHeadRevision  int64 = 1 // The resp.Header.Revision in last watch response.
```

Comment:
```suggestion
		lastEventModRevision int64 = 1
		lastResponseHeaderRevision  int64 = 1
```
Don't use comments for things easily expressed in name

Reply:
Why can't we use comment in code?

Changed `lastEventRevision` to `lastEventModRevision`.

Reply:
Adding comments should not replace good naming.

Reply:
The purpose of adding comment is **NOT** to replace good naming. 

Even experienced contributors may be confused about the two concepts (Header.Revision and event.ModRevision) and raise false alarm, you never know how other contributors (especially new contributors) read/understand your code, and what's the difference between them. It makes sense to me to add comment to clearly clarify what each of the two variables exactly means.

Note: I agree that we should give each variable/function/method a good name as possible as we can. Each event includes both `CreateRevision` and `ModRevision`, so changed `lastEventRevision` to `lastEventModRevision` to make it clearer.